tests/integration/models/reddit/test_redditor.py

```diff
@@ -103,6 +89,36 @@ def test_stream__submissions(self, _):
             for i in range(101):
                 assert isinstance(next(generator), Submission)
 
+    def test_trophies(self):
+        redditor = self.reddit.redditor('spez')
+        with self.recorder.use_cassette('TestRedditor.test_trophies'):
+            trophies = redditor.trophies()
+            assert len(trophies) > 0
+            assert len(trophies[0].name) > 0
+
+    def test_trophies__user_not_exist(self):
+        redditor = self.reddit.redditor('thisusershouldnotexist')
+        with self.recorder.use_cassette(
+                'TestRedditor.test_trophies__user_not_exist'):
+            with pytest.raises(BadRequest) as excinfo:
+                redditor.trophies()
+            response = excinfo.value.response
+            assert 'USER_DOESNT_EXIST' == response.json()['reason']
+
+    @mock.patch('time.sleep', return_value=None)
+    def test_unfriend(self, _):
+        self.reddit.read_only = False
+        with self.recorder.use_cassette('TestRedditor.test_unfriend'):
+            redditor = self.reddit.user.friends()[0]
+            assert redditor.unfriend() is None
+
+    @mock.patch('time.sleep', return_value=None)
```

Comment:
Thanks for re-ordering these to keep them sorted. It looks like unfriend and unblock, however, were previously, and still are in reverse order. Since you moved them, would you also mind correcting that relative order?